### PR TITLE
PD-38 - Add assets docs

### LINKED ISSUE #38: Feature: Document computation assets
### Is your feature request related to a problem? Please describe.

Assets have been added and need docs

### Describe the feature you are requesting, as well as the possible use case(s) for it.

Assets have been added and need docs

### Indicate the importance of this feature to you.

Must-have

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by WashingtonKK**
# What type of PR is this?

This is a feature documentation.

## What does this do?

This adds documentation for assets, a recently merged feature on prism.

## Which issue(s) does this PR fix/relate to?

Resolves #38

### CODE REVIEW

File: docs/assets.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Computation Assets Overview
+
+Computation assets on Prism provide a convenient way to manage essential computational resources—datasets and algorithms. These assets facilitate basic operations such as creation, reading, updating, and deletion (CRUD). They can also be associated with or unlinked from a computation.
```
Comment (SammyOina) [inline docs/assets.md:3]:
```suggestion
Computation assets on Prism provide a convenient way to manage essential computational resources including datasets and algorithms.  They can also be associated with or unlinked from a computation.
```

File: docs/assets.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+An important distinction is that computation assets contain file samples of the algorithm or dataset, providing users with a guide on what the actual files will resemble. These samples are **not** the actual files sent to the computation manifest when uploading via the CLI. For this reason, the actual file’s checksum is required during asset creation.
```
Comment (SammyOina) [inline docs/assets.md:5]: shared with collaborators in a computation

File: docs/assets.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+To create a new asset:
+
+1. Navigate to the **Assets** page.
+2. Click the **New Asset** button to open the asset creation modal.
+
+   ![Create asset](img/newAsset.png)
+
+The only mandatory fields during asset creation are:
+
+- **File Hash**: The checksum of the actual file (not the sample).
```
Comment (SammyOina) [inline docs/assets.md:22]: checksum and link to cli docs for this specific example

File: docs/assets.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- **Asset Description**: A brief summary of what the asset represents.
+
+The **File Name** field is optional. If provided, it will be used as the file name when running the computation (not the name of the sample file).
```
Comment (SammyOina) [inline docs/assets.md:25]: it will be verified when asset is uploaded on computation run

File: docs/assets.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+![View Asset](img/asset_view.png)
+
+### 4. **Running a Computation**
+
+Once all assets have been successfully associated with a computation, they will be listed within the computation’s structure. At this point, the user is required to upload their **public keys** to proceed.
+
+![Computation Page](img/asset_computation.png)
+
+After the public keys and other necessary requirements have been provided, the computation can be executed. During execution, the computation service will retrieve the associated assets from the database and add them to the computation manifest, which is then sent to the manager.
```
Comment (SammyOina) [inline docs/assets.md:67]: backend not manager

**Review by SammyOina — CHANGES_REQUESTED** [review]
include info on computation assets, the view on computation page

File: docs/assets.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ To create a new asset:
 
 The only mandatory fields during asset creation are:
 
-- **Checksum**: The checksum of the actual file (not the sample), get more information on checksum  [here](https://docs.cocos.ultraviolet.rs/cli/#file-hash).
+- **Checksum**: The checksum of the actual file (not the sample), get more information on checksum [here](https://docs.cocos.ultraviolet.rs/cli/#file-hash).
```
Comment (SammyOina) [inline docs/assets.md:22]: please rename to file-checksum

File: docs/assets.md [hunk 7]
```diff
@@ -59,7 +59,17 @@ If an asset sample has been uploaded, users can either view it directly on Prism
 
 ![View Asset](img/asset_view.png)
 
-### 4. **Running a Computation**
+### 4. **Viewing a computation's assets**
+
+To view assets associated to a computation, click the **Asset Management** which opens the assets page containing the computation's assets.
```
Comment (SammyOina) [inline docs/assets.md:64]: asset management button

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**